modules/engage-paella-player/gulpfile.js

```diff
@@ -24,39 +24,49 @@
 'use strict';
 var gulp = require('gulp');
 var spawn = require('child_process').spawn;
-var mergeStream = require('merge-stream');
+var request = require('request');
+var source = require('vinyl-source-stream');
+var gunzip = require('gulp-gunzip');
+var untar = require('gulp-untar');
 
+var PAELLA_VERSION = '6.3.2';
```

Comment:
@miesgre  This pull pushes the versio of JQuery, and other dependencies, back for people who have already built on this version of Paella. I recommend making this pull part part of the next Paella upgrade. Then the dependencies are as current or more current for the adopters when they upgrade.
https://github.com/polimediaupv/paella/pull/533

Reply:
In this PR I only change the way paella source code is downloaded
The idea is to update paella player to 6.4.2 once this PR was accepted, but I can include it in this commit if you want.

Reply:
I prefer the Paella update to coincide with this change because the update will be associated to a snapshot with the latest dependencies. Then there would not be the chance that an upgrade Opencast would give users older Paella dependencies.

Reply:
I updated this PR and to update paella to latest version 6.4.3 with newer dependencies (including jQuery 3.5.1)